### PR TITLE
Put auction

### PR DESCRIPTION
**Pull request by mikan373**
## What
GET request to update a bid price

## How
1. List an item from frontend
2. Check its uuid by getting items
```
curl -X GET http://localhost:9000/items
```
3. Use that uuid for POST request to post bid price
```
curl -X POST http://localhost:9000/auction/<uuid>
-d "items_id=<uuid>" -d "bid_price=5000"
```
4. Send a PUT request to update a bid price
```
curl -X PUT http://localhost:9000/auction/<uuid>  -d "bid_price="4000""
```

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -255,4 +256,89 @@ def update_status(id):
...
+        ''',
+        (item_id,)
+    )
+    response = { "items": [row for row in c] }
+    conn.close()
+
+    return response
+
+@app.put("/auction/{item_id}")
+def update_bid(item_id: str, bid_price: str = Form(...)):
```
Comment (mfunyu) [inline python/main.py:321]: Is `bid_price` str or int?

Comment (mfunyu) [inline python/main.py:321]: @dominic-soh 
Should update_bid check if `bid_price > price` or is that frontend's job?

Comment (dominic-soh) [reply python/main.py:321]: str shouldnt be an issue.

Comment (dominic-soh) [reply python/main.py:321]: I think i will make frontend validate this

**Review by dominic-soh — APPROVED** [review]

**Pull request merged**